doc/arch/adr-015-convention-for-data-warehouse-unique-id.md

```diff
@@ -0,0 +1,44 @@
+# Use compound id instead of a random UUID
+
+26-Sep-2018
+
+## Context
+
+In the Data Warehouse [we have decided to track content through time and use a unique identifier](https://github.com/alphagov/content-performance-manager/pull/924) that guarantees consistency across time.
```

Comment:
What about linking to the ADR its self rather then the PR? as it saves the user a click...

https://github.com/alphagov/content-performance-manager/blob/1f9f961897/doc/arch/adr-014-track-attribute-changes-per-basepath.md might

Reply:
Not sure why, but I always link to the PRs. Sorry. Good point.
Addressed in https://github.com/alphagov/content-performance-manager/pull/939/files#diff-fc9e31193c10d7d1347a6bfabcd2b441R7